docs/source/user_guide/installation.md

```diff
@@ -1,12 +1,19 @@
 # Installing pyRTE-RRTMGP
 
-This section contains instructions for installing pyRTE-RRTMGP, both for regular usage (with conda) and for development purposes (with pip).
+This section contains instructions for installing pyRTE-RRTMGP. On Linux (x86_64) platforms, you can install the package with `conda`. For other platforms, you need to build the package from source.
 
 ```{warning}
 This project is a work in progress. The Python API is not yet stable and is subject to change.
 ```
 
+## Installation
+
+```{include} ../../../README.md
+:start-after: <!-- start-installation-section -->
```

Comment:
Is this right? If I open the file I see the raw text.

Reply:
Thanks for checking! This is using MystMD, and Sphinx automatically inserts and renders the correct text here. At https://github.com/earth-system-radiation/pyRTE-RRTMGP/pull/72#discussion_r1869096169, you can see the result with the correctly rendered text blocks: 
![image](https://github.com/user-attachments/assets/60893c54-20df-413e-8f4c-3b6fd69937a4)